MADRaS/traffic/traffic.py

```diff
@@ -2,20 +2,25 @@
 from MADRaS.utils.gym_torcs import TorcsEnv
 import MADRaS.utils.snakeoil3_gym as snakeoil3
 from MADRaS.controllers.pid import PIDController
+import MADRaS.utils.madras_datatypes as md
 from multiprocessing import Process
+from time import time
 import logging
 logger = logging.getLogger(__name__)
 
+MadrasDatatypes = md.MadrasDatatypes()
+
 class MadrasTrafficManager(object):
     """Creates the traffic agents for a given training configuration."""
-    def __init__(self, torcs_server_port, cfg):
+    def __init__(self, torcs_server_port, num_learning_agents, cfg):
         self.traffic_agents = {}
```

Comment:
So this is an interesting issue.

This issue arose when `reset` was being called.
![cp1](https://user-images.githubusercontent.com/25224164/70615110-c8704e80-1c31-11ea-8e41-0091f5a93c88.png)

This seemed identical to the issue that was present in the `MultiAgent` part.
I tried this once `self.traffic_agents = OrderedDict()` and it works.

Reply:
Done.